viewer/packages/camera-manager/src/Flexible/ControlsType.ts

```diff
@@ -0,0 +1,12 @@
+/*!
+ * Copyright 2021 Cognite AS
```

Comment:
Can we have this be 2024 for all new files?

Reply:
Fixed